nixos/modules/security/hidepid.nix

```diff
@@ -33,6 +33,12 @@ in
   };
 
   config = mkIf cfg.enable {
+    assertions =
+     [ { assertion = cfg.procGroup != null -> (config.users.groups."${cfg.procGroup}" or "") != "";
+         message = "the group named at security.hideProcessInformation.procGroup does not exist!";
+       }
+     ];
+
```

Comment:
Sorry, I didn't understand at this point that you need GID (and need to resolve it somehow from the group name).